package.json

```
@@ -131,6 +131,7 @@
     "yargs": "^17.7.2"
   },
   "dependencies": {
+    "@cspell/dict-es-es": "^3.0.0",
```

```suggestion
    "@cspell/dict-es": "^3.0.0",
```
just use the generic code for spanish language and don't use the ES for region Spain